blog-posts/sls-security-headers/sls-security-headers.md

```diff
@@ -0,0 +1,96 @@
+---
+published: false
+title: 'Leverage AWS CloudFront managed response headers policies'
```

Comment:
You can find a more catchy title, like: 'Deliver perfect HTTP security headers with CloudFront`

Reply:
'Deliver perfect HTTP security headers with AWS CloudFront` (pour avoir un max de buzzwords dans le titre)